common/src/main/java/net/adoptopenjdk/icedteaweb/jvm/JvmUtils.java

```diff
@@ -301,9 +317,83 @@ public static String findOnPath(final String[] bins) {
                         break outerloop;
                     }
                 }
-
             }
         }
         return exec;
     }
+
+    /**
+     * https://docs.oracle.com/javase/9/tools/java.htm#JSWOR624
+     * Java 9 VM args. specified by app developer in jnlp file.
```

Comment:
This javaDoc should be updated/improved (if the method remains) as it still references java9 instead of java modules and the itw-modularjdk file

Reply:
ok